app/pb_kits/playbook/pb_icon_title/_icon_title.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= content_tag(:div, id: object.id,  data: object.data, class: object.classname(object.kit_class)) do %>
+
+<%= pb_rails("icon_circle", props: {
+  icon: "#{object.icon}",
+  size: object.column_sizing ,
+  variant:"#{object.icon_variant}"}) %>
+<div class="content">
+
+  <%= pb_rails("title", props: { text: object.value, size: object.size === "lg" ? 1 : 3 , dark: object.dark === "dark" ?  true : false }) %>
```

Comment:
Lets move this logic to the rb file

Reply:
Just the size logic. Also, the dark logic seem weird as it's declared as boolean everywhere and at some point the default value becomes a string. It would be cool if this component was moved to the new model already as this PR wasn't merged yet.